src/components/menu/RowContextMenu.tsx

```diff
@@ -34,13 +34,30 @@ const RowContextMenu: React.FC<RowContextMenuProps> = ({}) => {
     return false;
   }
 
+  function onCopyCellContent(p: ItemParams) {
+    const { props } = p;
+    const { rowIdx } = props;
+    const row = state.rows[rowIdx];
+    const columnKey = state
+      .gridColumns[state.selectedCellPosition?.idx as number]
```

Comment:
should have a guard clause as state.selectedCellPosition can be null

Reply:
I was of the thought that state.selectedCellPosition is updated when we right click on a cell, what's the instance where it can be null

Reply:
i don't have a use-case right now. but a safe check just makes your code better.